### ISSUE TITLE
Rename Repository

### ISSUE THREAD

**Issue by jatkinson1000** [GO-SHIP-Oceanography/station_bathymetry#7]
We should consider renaming this repo to be more 'active' as discussed at the GO-SHIP internal meeting.

"plot-bathymetry-at-station" was suggested?

**Comment by jatkinson1000** [comment]
@lcimoli Do you have any thoughts on this?

**Comment by lcimoli** [comment]
Agreed. plot-bathymetry-at-station sounds good to me (and I think it was what Alison suggested)
